blockly-core/core/ui/block_space/block_space.js

```diff
@@ -189,6 +194,12 @@ Blockly.BlockSpace.createReadOnlyBlockSpace = function (container, xml) {
   }, true, true);
 
   var blockSpace = blockSpaceEditor.blockSpace;
+
+  if (opt_options.noScrolling) {
+    blockSpace.scrollbarPair.dispose();
+    blockSpace.scrollbarPair = null;
+  }
```

Comment:
I was debating. It comes down to whether we view these ReadOnly BlockSpaces as a core feature deserving of refactoring to support, or if it's more of a curiosity we simply make allowances for.